sycl/source/detail/graph_impl.cpp

```diff
@@ -969,7 +969,8 @@ exec_graph_impl::enqueue(const std::shared_ptr<sycl::detail::queue_impl> &Queue,
               // TODO: Pass accessor mem allocations
               nullptr,
               // TODO: Extract from handler
-              PI_EXT_KERNEL_EXEC_INFO_CACHE_DEFAULT, CG->MKernelIsCooperative);
+              PI_EXT_KERNEL_EXEC_INFO_CACHE_DEFAULT, CG->MKernelIsCooperative,
+              CG->MKernelUsesClusterLaunch);
```

Comment:
What happens if we try to use this feature inside a SYCL-Graph? An exception should be thrown to say it's not supported (we'd need an equivalent of `piextEnqueueKernelLaunchCustom` in the UR command-buffer feature)

If you could add a test that an exception is thrown when using this feature in `sycl/unittests/Extensions/CommandGraph/Exceptions.cpp` it would be great

Reply:
@EwanC I have added the test, and a check in `setKernelUsesClusterLaunch` to throw an error if it's graph associated